.gitignore

```diff
@@ -1,12 +1,14 @@
-node_modules
+.bash_history
```

Comment:
Note: this commit is mislabeled, it's really a bugfix and general cleanup commit